example/http/client/client.go

```diff
@@ -58,11 +59,18 @@ func main() {
 	)
 
 	var body []byte
+	var url string
+
+	if len(os.Args) > 1 {
```

Comment:
Can we use the `flag` package here instead?

Reply:
Currently, `client` only accepts an optional argument, no flag processing:
```
$ ./client http://http-server:7777/hello
```

Are you suggesting `flag` package be used to pass in parameters as a flag instead of a command line argument? e.g.
```
$ ./client -u http://http-server:7777/hello
```

Reply:
Yeah, I was thinking along the lines of specifying an option instead of just an argument. That way help documentation it included in the CLI with a description of what you should pass in.

If an argument instead of an option is wanted, we could still update the help docs to say what that argument should be and what it does. I prefer to include documentation with CLIs so as the user doesn't have to go dig through the code to find out what to pass and how things are interpreted. The `flag` package is just a suggestion on how to accomplish that.

Reply:
@MrAlias i added git commit 7976c98 to resolve this change request